Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,68 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            float n1,n2,ans;
+            String temp1=null,temp2=null;
+            int p=equation.Length;
+            var x=equation.IndexOf('*');
+            var a=equation.Substring(0,x);
+            var y=equation.IndexOf('=');
+            //Console.WriteLine(y);
+            y=y-x-1;
+            var b=equation.Substring(x+1,y);

```

Comment:
you could have used var b=equation.Substring(x+1,y-x-1);
So that you dont need to calculate position of '=' again into y.